src/main/java/games/strategy/engine/ClientFileSystemHelper.java

```diff
@@ -118,7 +126,8 @@ public static File getUserMapsFolder() {
       try {
         mapsFolder.mkdirs();
       } catch (final SecurityException e) {
-        ClientLogger.logError(e);
+        ClientLogger.logError(
+            "Access denied, security exception while accessing user maps folder: " + path, e);
```

Comment:
I'm not sure if this error message is accurate.
According to the [javadocs](https://docs.oracle.com/javase/9/docs/api/java/io/File.html#mkdirs--) a SecurityException will only be thrown if there's a securitymanager active (by default there isn't) and this security manager explicitly denies the access to one of the directories.
It would probably be better to check if this method returns false instead.

Reply:
I'd rather not mess with existing logic, any suggestion for a better error message?

Reply:
Something like "A security manager denied the access to file/directory xyz"?